### PR TITLE
hide cookie banner if url passed to auto-decline

### PR DESCRIPTION
**Pull request by ILikeTeaALot**
does what it says on the tin really...

parameter name is "autoDeclineCookies"

it simply checks for the presence of the parameter, as it's name suggests a boolean value anyway

### CODE REVIEW

File: frontend/src/App.tsx [hunk 1]
```diff
@@ -120,8 +123,11 @@ class App extends Component<Record<string, never>> {
 						onDecline={
 							() => { gaSetState(true); window.location.reload(); }
 						}
+						// Due to the version of react-cookie-consent used missing the type declaration for the "visible" prop, this hack is sadly required.
+						/// @ts-expect-error
```
Comment (Gum-Joe) [inline frontend/src/App.tsx:127]: Linter error needs fixing (see error attached in the review panel)

Comment (ILikeTeaALot) [reply frontend/src/App.tsx]: file has been updated

**Review by Gum-Joe — CHANGES_REQUESTED** [review]
Rejected as non-functional.

**Review by Gum-Joe — APPROVED** [review]
Changes validated to work. Should see no issues build side.

**Pull request merged**